### ISSUE TITLE
tests.test_ling.test_glibc_supported('C.UTF-8',) failure

### ISSUE THREAD

**Issue by jwilk** [jwilk/i18nspector#13]
Seen on Ubuntu 22.04:
```
======================================================================
ERROR: tests.test_ling.test_glibc_supported('C.UTF-8',)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File ".../i18nspector/tests/test_ling.py", line 416, in t
    lang = L.parse_language(l)
  File ".../i18nspector/lib/ling.py", line 309, in parse_language
    raise LanguageSyntaxError
lib.ling.LanguageSyntaxError

----------------------------------------------------------------------
```

**Comment by jwilk** [comment]
```console
$ grep -w C /usr/share/i18n/SUPPORTED
C.UTF-8 UTF-8
```
This is weird, because `C.UTF-8` is a built-in locale on this system.

**Comment by jwilk** [comment]
Fixed in [0.27.1](https://github.com/jwilk/i18nspector/releases/tag/0.27.1).
